app/controllers/annotations_controller.rb

```diff
@@ -27,6 +34,14 @@ def create
   action_auth_level :update, :course_assistant
   def update
     @annotation.update(annotation_params)
+
```

Comment:
Annotation failures would be caught by @annotation.update returning false. Should case on that instead?